docs/integrations/aws.md

```diff
@@ -435,6 +435,26 @@ For the above example, the objects in S3 will be saved with tags: `my_key1=my_va
 
 For more details on tag restrictions, please refer [User-Defined Tag Restrictions](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html).
 
+### S3 Access Points
+
+[Access Points](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html) can be used for operating
+S3 objects from the specified bucket, against which access-point was mapped. This is useful for multi-region access, cross-region access,
```

Comment:
> can be used for operating S3 objects from the specified bucket, against which access-point was mapped. 

This sentence is a bit convoluted. What about "can be used to perform S3 operations by specifying a mapping of bucket to access points."